docs/guides/Integration-Guides/sent-offers-integration-guide-connectors.md

```diff
@@ -122,24 +111,13 @@ An interaction’s goal is to generate a process, or update to an offer. Once an
 
 This result is what allows Offer Management to infer an offer’s status. The types of interaction results, and their relation to offer statuses are the following:
 
-[block:parameters]
-{
-  "data": {
-    "h-0": "Interaction’s result",
-    "h-1": "Success",
-    "h-2": "Failure",
-    "h-3": "Notification",
-    "h-4": "Processing",
-    "0-1": "**Status: Synced** \n When closing a `success` interaction, Offer Management updates the offer’s status to `Synced`, if there aren't any active errors in this offer.\n\n\n**Status: Unavailable**\n When the offer already exists and the seller removes it from the trade policy, Offer Management doesn’t infer an offer’s status, and the connector indicates it. In this case, the connector must also remove the duplicate from the channel. Once the action is done, the interaction should be a success. \n\n In cases, the connector will send the `\"status\":\"unavailable\"` data in an interaction.",
-    "0-2": "**Status: Error**\nWhen an offer has at least one failure interaction, the offer’s status becomes `error`.",
-    "0-3": "**Status: Unavailable**\nWhen the offer is discarded by the connector - by being inactive or not included in the trade policy, Offer Management doesn’t infer an offer’s status, and the connector indicates it.  In that case, the connector will send the `\"status\":\"unavailable\"` data in a interaction.",
-    "0-4": "**Status: Sending** \nWhen the interaction is being processed, the offer’s status becomes `sending`when the interaction has the attribute “context”=”setup”.",
-    "0-0": "**Corresponding status** "
-  },
-  "cols": 5,
-  "rows": 1
-}
-[/block]
+| Interaction’s result | Corresponding status |
+|---|---|
+| Success | **Synced:** When closing a `success` interaction, Offer Management updates the offer’s status to `Synced`, if there aren't any active errors in this offer.
+||**Unavailable:** When the offer already exists and the seller removes it from the trade policy, Offer Management doesn’t infer an offer’s status. |
```

Comment:
Esse `||` faz com que o `Unavailable` entre em outra linha, quando você pode colocar na mesma só dando um line break `</br>` e removendo os `||`

Reply:
| Interaction’s result | Corresponding status |
|---|---|
| Success | **Synced:** When closing a `success` interaction, Offer Management updates the offer’s status to `Synced`, if there aren't any active errors in this offer. </br>**Unavailable:** When the offer already exists and the seller removes it from the trade policy, Offer Management doesn’t infer an offer’s status. |
| Failure | **Error:** When an offer has at least one failure interaction, the offer’s status becomes error. |
| Notification | **Unavailable:** When the offer is discarded by the connector - by being inactive or not included in the trade policy, Offer Management doesn’t infer an offer’s status, and the connector indicates it. In that case, the connector will send the "status":"unavailable" data in a interaction. |
| Processing | **Sending:** When the interaction is being processed, the offer’s status becomes sending when the interaction has the attribute “context”=”setup”. |

Reply:
```
| Interaction’s result | Corresponding status |
|---|---|
| Success | **Synced:** When closing a `success` interaction, Offer Management updates the offer’s status to `Synced`, if there aren't any active errors in this offer. </br>**Unavailable:** When the offer already exists and the seller removes it from the trade policy, Offer Management doesn’t infer an offer’s status. |
| Failure | **Error:** When an offer has at least one failure interaction, the offer’s status becomes error. |
| Notification | **Unavailable:** When the offer is discarded by the connector - by being inactive or not included in the trade policy, Offer Management doesn’t infer an offer’s status, and the connector indicates it. In that case, the connector will send the "status":"unavailable" data in a interaction. |
| Processing | **Sending:** When the interaction is being processed, the offer’s status becomes sending when the interaction has the attribute “context”=”setup”. |
```